thorin/world.h

```diff
@@ -145,8 +145,10 @@ class World {
     bool empty() { return move_.externals.empty(); }
     void make_external(Def* def) {
         assert(!def->name().empty());
-        auto [_, ins] = move_.externals.emplace(def->name(), def);
-        assert(ins);
+        auto name = def->name();
+        // assert(!move_.externals.contains(name));
+        auto [i, ins] = move_.externals.emplace(def->name(), def);
+        // assert((ins || (def == i->second)) && "make sure the external isn't inserted twice");
```

Comment:
we had the discussion in Discord. Can we enable the assertion again?

Reply:
We discussed that it is a problem and should not happen. But we did not resolve it.

Reply:
Moved to #99